tests/subsys/mgmt/zcbor_bulk/src/main.c

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include <sys/byteorder.h>
+#include <zcbor_common.h>
+#include <zcbor_decode.h>
+#include <zcbor_bulk/zcbor_bulk_priv.h>
+
+
+void test_correct(void)
+{
+	/* { "hello":"world", "one":1, "bool_value":true } */
+	uint8_t correct[] = {
+		0xA3, 0x65, 0x68, 0x65, 0x6C, 0x6C, 0x6F, 0x65, 0x77, 0x6F, 0x72, 0x6C, 0x64,
+		0x63, 0x6F, 0x6E, 0x65, 0x01, 0x68, 0x62, 0x6F, 0x6F, 0x6C, 0x5F, 0x76, 0x61,
+		0x6C, 0xF5
```

Comment:
```suggestion
		0x63, 'o', 'n', 'e', 0x01, 0x68, 'b', 'o', 'o', 'l', '_', 'v'. 'a'. 'l', 0xF5,
```

Reply:
To be honest I have used cbor.me to generate these and piped result through sed to make it array of hex.

Reply:
I see. You could also use zcbor_encode.h to build the payload if you find that more convenient.